assets/prometheus-adapter/config-map.yaml

```diff
@@ -1,5 +1,21 @@
 apiVersion: v1
 data:
+  audit-policy-metadata.yaml: |-
```

Comment:
@dgrisonnet if we are happy with the approach, I will submit the jsonnet changes to `prometheus-adapter.libsonnet`

Reply:
I am bit lost here, 
* Where would the `policy object` be stored?  IIUC we need an actual audit-policy `file` that can be read by the api server. 
* Are you suggesting that CMO generate the policy file at runtime, create a config-map  and then inject that into the promethes-adapter (update the deployment), if so, I am not sure what advantage it gives us 🤔

Reply:
As per our offline discussion, we should stick to having a configmap containing all policy manifests, but it should be separated from the prometheus-adapter config.